migrations/sqlite/20240809205107_event_source_anchored.up.sql

```diff
@@ -0,0 +1,12 @@
+-- Add up migration script here
+
+-- The CID of the Init Event for the stream
+ALTER TABLE ceramic_one_event ADD COLUMN init_cid BLOB DEFAULT NULL;
```

Comment:
how will this get backfilled for events that were created before this field was present in the db?

Really ideally this column would be non-nullable and always required, but that makes the upgrade path trickier

Reply:
Right now we're only using this field while generating Time Events, which will only happen for events that also have the `informant` populated. We should never have events with an `informant` and without a `init_cid`. This means that the `init_cid` being missing is ok from the self-anchoring perspective, but it does still need to be populated so that we can support things like looking up events for a stream. 

Created a [ticket](https://linear.app/3boxlabs/project/self-anchoring-pt-1-157039361486/issues) for this.